### ISSUE TITLE
Why not using dual Quaternions for translations and rotations?

### ISSUE THREAD

**Issue by DadoooR3** [KieranWynn/pyquaternion#34]
im newbie in the quaternions world. I found our library usefull.
I'm searching solutions for translations and rotations problems.
I found this paper over dual quaternions. Are you interested for development of these functionalities in our library?

[dual-quaternion.pdf](https://github.com/KieranWynn/pyquaternion/files/3025602/dual-quaternion.pdf)

Bests regards.

Daniel

**Comment by Hojjatrt** [comment]
hey man read a few of that let's implements that

**Comment by KieranWynn** [comment]
I agree this would be a nice feature to add. I'll add it to the roadmap.

**Comment by Achllle** [comment]
Here's a repo for that: [dual-quaternions](https://github.com/Achllle/dual_quaternions_ros). It currently uses `numpy-quaternion` as its backend and can be installed via `pip install dual-quaternions`, but I'm [working on switching to `pyquaternion`](https://github.com/Achllle/dual_quaternions_ros/tree/pyquaternion_switch) because it has fewer dependencies so it can distributed into `apt`.
